src/app/admin/admin-reports/filtered-items/filtered-items.component.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { waitForAsync, ComponentFixture, TestBed} from '@angular/core/testing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { TranslateLoader, TranslateModule, TranslateService } from '@ngx-translate/core';
+import { TranslateLoaderMock } from 'src/app/shared/mocks/translate-loader.mock';
+import { FormBuilder } from '@angular/forms';
+import { FilteredItemsComponent } from './filtered-items.component';
+import { DspaceRestService } from 'src/app/core/dspace-rest/dspace-rest.service';
+import { NgbAccordion, NgbAccordionModule } from '@ng-bootstrap/ng-bootstrap';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { of as observableOf } from 'rxjs';
+import { RawRestResponse } from 'src/app/core/dspace-rest/raw-rest-response.model';
+import { CommunityDataService } from 'src/app/core/data/community-data.service';
+import { ObjectCacheService } from 'src/app/core/cache/object-cache.service';
+import { ActionsSubject, ReducerManager, ReducerManagerDispatcher, StateObservable, Store, StoreModule } from '@ngrx/store';
```

Comment:
This line has lint errors because you don't use ActionsSubject, ReducerManager, ReducerManagerDispatcher, StateObservable or Store in this spec. They should all be removed from this import